### PR TITLE
Putting Tricord pill bottles in med belts

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

Title.

Also removing tricord pill bottles that cost 8 corpsman points as option to buy in corpsman vendor.

## Why It's Good For The Game

Currently, you can get tricord pill bottles for free in MarineMed. They stock 12 in round start. This means that any early-spawn marines, including early-spawn corpsmen, can get it, and usually early-spawn corpsmen will yoink one for themselves since it's a good T1 med.

When the round progresses and passes the 13:00 mark, it is virtually out of stock, and late-spawn corpsmen that want a tricord pill bottle would have to buy one from corpsman vendor or make it themselves.

Kinda wack for late-spawn corpsmen to deal with that.

## Changelog
:cl:
add: tricord pill bottles in all medical belts, whether med rig or lifesaver
del: option to buy tricord pill bottle in corpsman vendor
qol: corpsmen can get tricord pill bottle in belt
balance: tricord pill bottle round start
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -137,14 +137,14 @@
...
 	new /obj/item/storage/pill_bottle/tramadol(src)
+	new /obj/item/storage/pill_bottle/tricordrazine(src)
 	new /obj/item/storage/pill_bottle/dylovene(src)
 	new /obj/item/storage/pill_bottle/inaprovaline(src)
 	new /obj/item/storage/pill_bottle/dexalin(src)
 	new /obj/item/reagent_containers/hypospray/autoinjector/peridaxon_plus(src)
 	new /obj/item/reagent_containers/hypospray/autoinjector/quickclotplus(src)
 	new /obj/item/storage/pill_bottle/spaceacillin(src)
 	new /obj/item/storage/pill_bottle/hypervene(src)
-	new /obj/item/healthanalyzer(src)
```
Comment (novaepee) [inline code/game/objects/items/storage/belt.dm:147]: I have to admit, this is a bit iffy doing this, but corpsman has health analyzer in essential kit anyway.

Comment (jroinc) [reply code/game/objects/items/storage/belt.dm:147]: I'd do the spaceacillin instead, if anything. Honestly, I'd leave it out of the medbelt, but that's your call.

Comment (novaepee) [reply code/game/objects/items/storage/belt.dm:147]: In another headspace, I would replace spaceacillin, though infection control is important (and currently infection is at an all time high for some reason).

Comment (BraveMole) [reply code/game/objects/items/storage/belt.dm:147]: yeah kill spacecilin

**Review by BraveMole — CHANGES_REQUESTED** [review]
Also you can touch the other belt here

**Review by BraveMole — APPROVED** [review]

**Comment by JanStruan** [comment]
I would like to preface this by saying my opinion doesnt count and I came from no where, but this seems like a thing that Tanya complained about once as a synth. Medics can deal with necro pretty often, and so need Spacilin. Also tricord as a chem is a poor medics chem. It does heal every basic damage, but it does it slowly. A medics true purpose is not to FULLY HEAL marines but get them back to "fully mission capable" as quickly as possible. However that definition of "fully mission capable" is subject to an individual's interpretation. Marines are just a piece of a cog to the Marine Corps machine.

**Pull request merged**